modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/template/RESTClientTemplateContextContributor.java

```diff
@@ -59,6 +61,17 @@ public RESTClient(
 		}
 
 		public Object get(String path) throws Exception {
+			try {
+				return _get(path);
+			}
+			catch (Throwable throwable) {
+				_log.error(throwable, throwable);
```

Comment:
Which is the difference between `_log.error(throwable, throwable);` and `_log.error(throwable);`. I have seen both patterns. Just to decide which one we want to leave

Reply:
The first form includes the stack trace; the second will only log the message. The first form is preferred IMHO.

Reply:
But I see two methods with a similar header:

	public void error(Object msg);

	public void error(Throwable throwable);

Maybe it was just a matter of which one was being used by Java?

Reply:
Sure, but the implementation is the first logs msg.toString() while the second logs throwable.getMessage().

Either way, getting the stack trace is important for debugging purposes.

Reply:
I just made the test locally by using both methods and at least in my case, both prints the whole stack trace. I guess the method with the message + throwable is available to print, one the one hand, the desired message and then, the whole stacktrace, while for the throwable case, we have no message to print apart from the whole stacktrace. That was the purpose of my comment